internal/schemas.ts

```diff
@@ -810,7 +810,7 @@ const CompressedVideo: FoxgloveMessageSchema = {
       name: "data",
       type: { type: "primitive", name: "bytes" },
       description:
-        "Compressed video frame data. For packet-based video codecs this data must begin and end on packet boundaries (no partial packets), and must contain enough video packets to decode exactly one image (either a keyframe or delta frame).",
+        "Compressed video frame data. For packet-based video codecs this data must begin and end on packet boundaries (no partial packets), and must contain enough video packets to decode exactly one image (either a keyframe or delta frame). NOTE: It is strongly recommended to _not_ write B frames/slices to the H264 stream and instead solely use I and P frames because this more closely mirrors live data sources.",
```

Comment:
a B frame is technically a "delta" frame. What about something like:
```suggestion
        "Compressed video frame data. For packet-based video codecs this data must begin and end on packet boundaries (no partial packets), and must contain enough video packets to decode exactly one image (either a keyframe or delta frame). B-frames should be avoided because of their incompatibility with live sources, due to them referencing a future frame.",
```

Reply:
I think "live sources" is a bit unclear and it's also not obvious why someone who's just writing an mcap file should care about live sources

Reply:
I think a more accurate answer is because of their incompatibility with streaming playback. Displaying B-frame encoded video requires lookahead.